pyat/at/lattice/options.py

```diff
@@ -30,6 +30,7 @@ class _Dst(object):
     OrbMaxIter = 20          # Max. number of iterations for orbit
     omp_num_threads = int(os.environ.get('OMP_NUM_THREADS', '0'))
     patpass_poolsize = multiprocessing.cpu_count()
+    patpass_startmethod = None
```

Comment:
Could you document the new `patpass_startmethod` option in the docstring of `DConstant`, a few lines later?

Reply:
Done